wasabi/printer.py

```diff
@@ -127,6 +131,8 @@ def text(
             title = wrap(title, indent=0)
         if text:
             title = "{}\n{}".format(title, wrap(text, indent=0))
+        if self.timestamp:
+            title = "{}\t{}".format(datetime.datetime.now(), title)
```

Comment:
Maybe we can trim the timestamp up to seconds? I think it's something like `datetime.datetime.now().strftime("%H:%M:%S")`?

Reply:
You still want the date though, right?

Reply:
Ah, good question... I missed this. Should we have the date? Do timestamps like this need dates? Do they typically have dates? I just realised I never pay attention to this because I only really look at the time.

Reply:
I usually do add the date, yes. I like my logs having dates on them, it helps me keep track of when different experiments were run. But also to support runs that may take a few days to complete, I think it's nicer. Could also add in an additional parameter (`no_date` ?) but that feels like overkill ?